tests/ro/creator/data/roc-02-emobon-observatory.yml

```
@@ -0,0 +1,71 @@
+    roles: [ contact, maintainer]
+  "https://edmo.seadatanet.org/report/422": # VLIZ
+    roles: provider
+  "https://edmo.seadatanet.org/report/429": # UGENT
+    roles: provider
+  "https://edmo.seadatanet.org/report/432": # RBINS
+    roles: provider
+  "https://edmo.seadatanet.org/report/2248": # KUL
+    roles: provider
+  # last agents should be possible to imply/inject from emobon-strategy, 
+  # no need to duplicate here
+  EMOBON: # EMOBON
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)